web/src/SidebarResources.tsx

```diff
@@ -129,6 +123,65 @@ let SidebarItemTimeAgo = styled.span`
 type Resource = Proto.webviewResource
 type Build = Proto.webviewBuildRecord
 
+export const triggerUpdate = (name: string): void => {
+  let url = `/api/trigger`
+
+  fetch(url, {
+    method: "post",
+    body: JSON.stringify({
+      manifest_names: [name],
+      build_reason: 16 /* BuildReasonFlagTriggerWeb */,
+    }),
+  }).then(response => {
+    if (!response.ok) {
+      console.log(response)
+    }
+  })
+}
+
+let SidebarItemLinkButton = styled.button`
+  display: flex;
+  align-items: stretch;
+  text-decoration: none;
+  // To truncate long names, root element needs an explicit width (i.e., not flex: 1)
+  width: calc(100% - ${Width.sidebarTriggerButton}px);
+  background: transparent;
+  border: none;
+  color: inherit;
+  font: inherit;
+  padding: 0;
+  margin: 0;
+`
+
+let SidebarItemLinkAnchor = styled(Link)`
+  display: flex;
+  align-items: stretch;
+  text-decoration: none;
+  // To truncate long names, root element needs an explicit width (i.e., not flex: 1)
+  width: calc(100% - ${Width.sidebarTriggerButton}px);
+`
+
+function SidebarItemLink(props: {
+  title: string
+  onTrigger?: () => void
+  onNavLink: string
+  isSelected: boolean
+  children: any
+}) {
+  if (props.isSelected && props.onTrigger) {
+    return (
+      <SidebarItemLinkButton onClick={props.onTrigger} title={props.title}>
```

Comment:
when hovering over a selected sidebar item, i do NOT get the pointing-hand cursor, that seems wrong if they're clickable now?

Reply:
fixed!